src/lib/api-types.d.ts

```diff
@@ -202,6 +202,22 @@ export type Status = {
   poll: Poll | null
 }
 
+type UpdateCredentialsParams = {
```

Comment:
nistet of using `?:` everywhere you could use Typesceipts utility type `Partial<T>`

Reply:
This is awesome! I didn't know TypeScript had something like this